src/util/ConstexprUtils.h

```diff
@@ -235,6 +235,11 @@ constexpr void forEachTypeInParameterPack(const auto& lambda) {
   (lambda.template operator()<Ts>(), ...);
 }
 
+template <typename... Ts>
```

Comment:
Add a short comment (also for the helpers below)
@elB4RTO You don't have to write the comment, I will do this before merging.